mono/mini/debugger-engine.c

```diff
@@ -1022,8 +1037,6 @@ mono_de_process_breakpoint (void *void_tls, gboolean from_signal)
 	SeqPoint sp;
 	gboolean found_sp;
 
-	if (rt_callbacks.try_process_suspend (tls, ctx))
-		return;
```

Comment:
I don't understand this change.

Reply:
I changed the behavior of try_process_suspend, I think that now it's more clear the change.